ogn/parser/pattern.py

```diff
@@ -37,12 +38,15 @@
     RAM:(?P<ram_free>[\d.]+)\/(?P<ram_total>[\d.]+)MB\s
     NTP:(?P<ntp_offset>[\d.]+)ms\/(?P<ntp_correction>[+-][\d.]+)ppm\s?
     (?:(?P<cpu_temperature>[+-][\d.]+)C\s*)?
+    (?:(?P<aircraft_counter_visible>\d+)\/(?P<aircraft_counter_total>\d+)Acfts\[1h\]\s*)?
```

Comment:
Alrighty! Well, you're not yet parsing the RF-block, my code on the other hand doesn't know about the latest extensions. Where have you found that the new numbers refer to aircraft counts? (I still haven't found a reliable reference implementation or protocol changelog, have you?)